uisdk/src/main/res/values-it/strings.xml

```diff
@@ -348,7 +348,7 @@
     <string name="kh_uisdk_loyalty_unknown_error">Si è verificato un errore. Per guadagnare o utilizzare punti Reward, riprova più tardi.</string>
     <string name="kh_uisdk_loyalty_unsupported_currency">Errore: questa valuta non è supportata</string>
     <string name="kh_uisdk_loyalty_use_points_off_subtitle">Puoi usare punti Reward per pagare questa corsa</string>
-    <string name="kh_uisdk_loyalty_use_points_on_subtitle">Usa %s punti Reward per pagare %s</string>
+    <string name="kh_uisdk_loyalty_use_points_on_subtitle">Usa  %3$s punti Reward per pagare %1$s  %2$s</string>
```

Comment:
This seems to have some extra spaces before placeholders 2 and 3. Should be fixed in Phrase and re-synced